frontend/packages/dev-console/src/components/pipelineruns/PipelineTaskStatus.scss

```
@@ -0,0 +1,27 @@
+@import '../pipelines/PipelineStatusColors.scss';
+
+.odc__task__status {
+  .tooltip__legend {
+    height: 1rem;
+    width: 1rem;
+    margin-bottom: 0.5rem;
+  }
+  .tooltip--text {
+    text-align: left;
+  }
+  .tooltip--col {
+    padding-left:  var(--pf-global--spacer--sm);
+  }
+  .bg--running {
```

You can use the short modifier format although i think you're better off applying the colors directly in code using the same patternfly react-token values used by the charts.
```suggestion
  &.is-running {
```